tests/NEO/AccountTests.cpp

```diff
@@ -0,0 +1,25 @@
+// Copyright © 2017-2019 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#include "Hash.h"
+#include "HexCoding.h"
+#include "PrivateKey.h"
+#include "NEO/Signer.h"
+
+#include <gtest/gtest.h>
+
+using namespace TW;
+using namespace TW::NEO;
+
+TEST(NEOAccount, validity) {
+    auto hexPrvKey = "4646464646464646464646464646464646464646464646464646464646464646";
+    auto hexPubKey = "031bec1250aa8f78275f99a6663688f31085848d0ed92f1203e447125f927b7486";
+    auto signer = Signer(PrivateKey(parse_hex(hexPrvKey)));
+    auto prvKey = signer.getPrivateKey();
```

Comment:
let's remove these methods and test from Signer

Reply:
done